src/components/file/file.service.ts

```diff
@@ -103,25 +107,50 @@ export class FileService {
   ): Promise<File> {
     try {
       // TODO find a better way to check if object exists in s3 and move
+      let user: User | null;
+      const userSession: ISession = await this.authService.decodeAndVerifyToken(session.token);
+      user = (userSession.userId) ? await this.userService.readOne(userSession.userId, session) : null;
+
       const result = await this.db
         .query()
         .raw(
           `
-        MATCH (token:Token {active: true, value: $token})
+        MATCH
+        (token:Token {
+          active: true,
+          value: $token
+        })
+        <-[:token {active: true}]-
+        (requestingUser:User {
+          active: true,
+          id: $requestingUserId
+        })
         CREATE
-            (file:FileNode { id: $id, type: $type, name: $name })
+          (file:FileNode {
+            id: $id,
+            type: $type,
```

Comment:
these values should not be properties on the node.  If you look at the database schema.jpg, you will see that these are property nodes.  The create function in property-updater service will do this for you.  you just have to pass in the key/values into the function